serde/se.py

```diff
@@ -174,6 +174,7 @@ def wrap(cls: Type):
         g['typename'] = typename  # used in union functions
         g['is_instance'] = is_instance  # used in union functions
         g['to_obj'] = to_obj
+        g['Optional'] = Optional
```

Comment:
I feel like we should import `typing` module into the scope instead of `Optional`, `Union` ..etc